README.md

```diff
@@ -138,8 +139,8 @@ nearly instantaneous.
 You can test UTA and seqrepo installations like so:
 
 ```shell
-$ psql -XAt postgres://anonymous@localhost/uta -c 'select count(*) from transcript'
-249909
+$ psql -XAt postgres://anonymous@localhost/uta -c 'select count(*) from uta_20210129b.transcript'
```

Comment:
Does this connection string need to include the password added here?

https://github.com/ga4gh/vrs-python/pull/405/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R27

Reply:
@theferrit32 I was able to run the command without the password included

Reply:
@theferrit32 I _think_ it's because anonymous user doesn't have a password and that's why it's not needed

Reply:
@korikuzma got it, just tried it on my laptop and it also worked